components/ui/copied-message/copied-message.composition.tsx

```diff
@@ -4,20 +4,12 @@ import { ThemeCompositions } from '@teambit/documenter.theme.theme-compositions'
 import { Icon } from '@teambit/evangelist.elements.icon';
 import { H5 } from '@teambit/documenter.ui.heading';
 
-const codeString = `export function Anchor(props: AnchorProps) {
-  return (
-    <a {...props} id={props.href} href={href} className={styles.anchor}>
-      <Icon of="anchor" className={styles.anchorIcon}></Icon>
-    </a>
-  );
-}`;
-
 export const CopiedMessageExample = () => {
   return (
-    <ThemeCompositions>
-      <div style={{ position: 'relative' }}>
-        <CopiedMessage show={true} />
-      </div>
+    <ThemeCompositions
+      style={{ position: 'relative', paddingLeft: 50, height: 20 }}
+    >
```

Comment:
check this is working preperly. I remember I had issues with it, thats why I added the div